include/athena/operation/GEMMOperation.h

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright (c) 2018 Athena. All rights reserved.
+ * https://getathena.ml
+ *
+ * Licensed under MIT license.
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an “AS IS” BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+#ifndef ATHENA_GEMMOPERATION_H
+#define ATHENA_GEMMOPERATION_H
+
+#include <athena/core/operation/Operation.h>
+#include <athena/operation/internal/GEMMOperationInternal.h>
+#include <athena/operation/operation_export.h>
+
+namespace athena::operation {
+class ATH_OPERATION_EXPORT GEMMOperation : public core::Operation {
```

Comment:
I think it sounds better this way.
```suggestion
class ATH_OPERATION_EXPORT MatmulOperation : public core::Operation {
```